.github/workflows/build.yml

```
@@ -0,0 +1,122 @@
+  e2e:
+    runs-on: [self-hosted, Linux, X64, praevis-web]
+    needs: build
+
+    services:
+      backend:
+        image: praevis-api:latest
+        env:
+          ASPNETCORE_ENVIRONMENT: Test
+          ConnectionStrings__DefaultConnection: "Host=postgres;Database=praevis_test;Username=postgres;Password=postgres"
+        ports:
+          - 5000:80
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)